src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -182,12 +179,9 @@ where
 
     let mut tokens = vec![];
 
-    let (source_upper_tx, source_upper_rx) = config.metrics.get_instrumented_source_channel(
-        config.id,
-        config.worker_id,
-        config.worker_count,
-        "source_upper",
-    );
+    let (ingested_upper_tx, ingested_upper_rx) =
+        watch::channel(MutableAntichain::new_bottom(C::Time::minimum()));
+    let (_probed_upper_tx, probed_upper_rx) = watch::channel(None);
```

Comment:
This `probed_upper_tx` will be connected to the background upper checks in sources in the next PR